workers/generic-worker/os_groups_insecure_test.go

```diff
@@ -22,20 +21,15 @@ func TestEmptyOSGroups(t *testing.T) {
 	_ = submitAndAssert(t, td, payload, "completed", "completed")
 }
 
-func TestNonEmptyOSGroups(t *testing.T) {
+func TestNonEmptyOSGroupsUserIsNotIn(t *testing.T) {
 	setup(t)
 	payload := GenericWorkerPayload{
 		Command:    listGroups(),
 		MaxRunTime: 30,
-		OSGroups:   []string{"abc"},
+		OSGroups:   []string{"docker"},
```

Comment:
it is quite feasible that the user will be in the docker group on some worker pool, I think a more abstract fake group like "abc" is indeed better here.